### DISCUSSION TITLE
中大通識修課經驗(3)

### DISCUSSION THREAD

**Discussion by THH1118** [1101-GS4719-C/Python-Community-GS4719-C#1695] [General]
大二下

資料蒐集與報告寫作(朱碧靜老師)

這堂課是蠻注重分組的一個課程，你的分組會很大程度的影響你的修課體驗以及你的分數，所以簡單說一下分組方式:前三堂課是自己分組，通常就是跟自己附近的就近找，然後老師會給議題做小組討論，第四堂課老師就會給每個人一個許願卡，可以自己寫希望組員有甚麼特質，甚至可以直接寫自己希望跟誰一組，當然也需要寫自己具有甚麼特質，然後老師就會依照每個人的許願卡分組，然後之後的很多時間都會跟組員一起合作，期末也需要一起完成一份報告，然後還有自己要做一份簡單的書面報告作業，整體來說很適合喜歡小組合作的人，然後老師偶爾會有可以加分的機會，給分的話我自己是每個都有認真做，組員也還不錯，最後拿到的分數蠻高的。

大三下

影像紀錄製作基礎課程(黃文珊老師)

首先這堂課開在禮拜五早八到早十哈哈，每次上課都會點名但我們這屆助教都會讓我們補點名，所以基本上很多人都9點才來，每個禮拜老師會給一個主題，要根據那個主題來拍照片(大約十到十二張)，然後每個禮拜隨機抽人上去報告自己拍的照片，所以只要你沒被抽到那個禮拜基本上就是去聽下禮拜作業要交什麼而已，期中期末考也是給主題拍照，按照點名順序上台報告，這裡就要講一個這堂課可怕的地方，就是你的成績是由其他同學來決定的，老師會發評分表給同學評分，你的分數就是所有人給你的平分平均起來，所以如果你很認真準備但下面的人沒在聽隨便給分的話就會很燥，雖然看起來老師是也有在寫評分表但真的不知道那個成績是怎麼加總的，雖然我自己最後的成績蠻高的，但我也有同學不同學期修拿的成績普通，因此要修的話要稍微有點心理準備，我到總成績出來的前一刻都沒有頭緒我自己會拿幾分哈哈。

**Comment by mrweixd** [comment]
推資料蒐集與報告寫作，只要討論完就可以走人，很常提早下課。
